gdp/column/column.py

```diff
@@ -543,151 +743,250 @@ def YP_or_notYP(m, n):
+
+    Return:
+        None: The function adds the constraints directly to the model object and does not 
+        return a value.
+    """
+    # Defining the tray as the condenser tray for the mass balance
     t = m.condens_tray
 
+    # This function defines the mass balance for each component in the condenser tray
     @m.Constraint(m.comps)
     def condenser_mass_balance(_, c):
+        """Mass balance for each component in the condenser tray."""
         return (
-            - m.V[c, t]      # Vapor from tray t
+            - m.V[c, t]      # Vapor leaving the tray
             - m.D[c]         # Loss to distillate
-            - m.L[c, t]      # Liquid to tray below
-            + m.V[c, t - 1]  # Vapor from tray below
+            - m.L[c, t]      # Liquid flowing to the tray below
+            + m.V[c, t - 1]  # Vapor coming from the tray below
             == 0)
 
     @m.partial_cond.Constraint(m.comps)
     def condenser_liquid_composition(_, c):
+        """This is the composition of the liquid phase in the condenser tray for partial condensation"""
         return m.L[c, t] == m.liq[t] * m.x[c, t]
 
     @m.partial_cond.Constraint(m.comps)
     def condenser_vapor_composition(_, c):
+        """This is the composition of the vapor phase in the condenser tray for partial condensation"""
         return m.V[c, t] == m.vap[t] * m.y[c, t]
 
     @m.total_cond.Constraint(m.comps)
     def no_vapor_flow(_, c):
+        """This function ensures no vapor flow for each component in the case of total condensation."""
         return m.V[c, t] == 0
 
     @m.total_cond.Constraint()
     def no_total_vapor_flow(_):
+        """This function ensures no total vapor flow in the case of total condensation."""
         return m.vap[t] == 0
 
     @m.total_cond.Constraint(m.comps)
     def liquid_fraction_pass_through(_, c):
+        """This function ensures the mole fractions of the liquid phase are equal to those of the vapor phase from the tray below"""
         return m.x[c, t] == m.y[c, t - 1]
 
     @m.Constraint(m.comps)
     def condenser_distillate_composition(_, c):
+        """This function defines the composition of the distillate in the condenser tray"""
         return m.D[c] == m.dis * m.x[c, t]
 
 
 def _build_reboiler_mass_balance(m):
+    """
+    Constructs the mass balance equations for the reboiler tray in a distillation column.
+    This includes the definition of constraints for mass balance, and liquid and vapor composition.
+    The function considers the molar flow rates to establish the mass balance in the reboiler.
+
+    Args:
+        m (Model Object): A model object that includes the relevant variables, parameters, 
+        and expressions for the distillation process, such as component flows, liquid 
+        and vapor compositions.
+
+    Return:
+        None: The function adds the constraints directly to the model object and does not 
+        return a value.
```

Comment:
Are you including the constrains as above?

Reply:
Added the constraints and the explanation on the top of the documentation.